aspnetcore/blazor/advanced-scenarios.md

```diff
@@ -37,7 +37,7 @@ In <xref:Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder> methods wi
 
 ```razor
 @page "/built-content"
-@attribute [RenderModeServer]
+@rendermode RenderMode.InteractiveServer
```

Comment:
This is fine for now, but I think we'll want to switch to the shorter form (`@rendermode InteractiveServer`) once we ship the updated template with the `using static` directive for `RenderMode`.